template/Dockerfile

```diff
@@ -11,4 +11,5 @@ RUN npm install --production
 
 COPY . .
 
-CMD ["npm", "start"]
\ No newline at end of file
+EXPOSE 3000
```

Comment:
It's not necessary. Please remove it.

Reply:
As above discussed, that is necessary due to traefik v2 changed. Refer https://github.com/moleculerjs/moleculer-template-project/pull/15#issuecomment-544258145

You could try to test by yourself, create new template then remove that line. It just tried again, remove `expose` didn't work.
```
moleculer init jellydn/moleculer-template-project#patch-1 test-module
```

Reply:
But my problem with it that the port of API gateway is configurable inside docker-compose with `PORT: 3000`. So if someone wants to change, it won't work because in Dockerfile it is static.

Reply:
By the way, I think other options maybe incorrect because it works only from localhost. The v1.7 works from remote hosts, as well.

Reply:
In that sense, we could stick to v1.7 for awhile until we found a better solution.